documentation/CAMARA-API-Event-Subscription-and-Notification-Guide.md

```diff
@@ -335,7 +337,40 @@ For consistency across CAMARA APIs, the uniform CloudEvents model must be used w
 
 Note: For operational and troubleshooting purposes it is relevant to accommodate use of `x-correlator` header attribute. API listener implementations have to be ready to support and receive this data.
 
-### 3.2. `subscription-ends` event
+### 3.2. `subscription-begins` event
+
+Specific event notification type "subscription-begins" is defined to inform listener about subscription initiation.
+
+It is used when subscription creation request (POST /subscriptions) is managed asynchronously. For this specific event, the `data` must feature `initiationReason` attribute. 
+
+The following table lists values for `terminationReason` attribute:
```

Comment:
Should be` initiationReason` and not `terminationReason` probably

Reply:
have also fixed some wording and typos